### DISCUSSION TITLE
Is anyone experiencing issues with dictation on ios?

### DISCUSSION THREAD

**Discussion by transcendr** [Flutter-Bounty-Hunters/super_editor#882] [Q&A]
We are getting this behavior on ios only where you can dictate to the document at any length without issue until you add a new line.  After this point, any dictated text is deleted immediately after insertion.  Before assuming it's a SE issue, I was wondering if anyone else has seen this, and what I might be able to do debug and determine whether indeed it is a SE issue or something else. I'm not able to reproduce this behavior on Android or web.

https://user-images.githubusercontent.com/37011672/206189418-860c238c-22ee-4112-b445-3ff3ff26bda0.mov

**Comment by daverad** [comment]
This made me think: is there a list of live apps using SuperEditor? Could be a handy way for us to test if an issue is app specific, and also see (and celebrate!) what others in the SE community are building!

**Comment by matthew-carroll** [comment]
Please be sure to file issues for specific problems. We don't monitor discussion threads for work items.

**Comment by transcendr** [comment]
@matthew-carroll Thank you for your response and the reminder. I typically do file issues, but in this case, I wanted to see if anyone had similar experiences or could provide information about debugging to help determine if this is an SE editor issue or something else. I understand that issues may not always get a lot of feedback, so I wanted to ask the community in this discussion thread. In the future, I will make sure to file an issue for specific problems that I'm certain are SE issues.
